### DISCUSSION TITLE
Please enable high fps

### DISCUSSION THREAD

**Discussion by lokastoryid** [rustdesk/rustdesk#8232] [General]
If possible please enable high fps like 60fps or 144,because it just 30 fps. It will helpful for gamers and streamer thankyou

**Comment by 21pages** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/14891774/67a36b81-3e70-4de0-b410-d2f47c4c64c0)

**Reply by lokastoryid** [reply]
do yo know how to do in android?

**Reply by xlionjuan** [reply]
@lokastoryid same icon

**Comment by txbxxx** [comment]
Why am I turned on and it's still at 30fps?

**Reply by xlionjuan** [reply]
Use H265 or H264 codec

**Reply by penguinsam** [reply]
Any other factors that blocking higher FPS? I already changed to 120 FPS in custom and H265 codec, but the highest FPS seems locked up to around 60 FPS.

Currently using direct IP connect in local LAN and I have my own ID server.

**Reply by xlionjuan** [reply]
Use direct IP to connect, no server needed

**Reply by penguinsam** [reply]
Yes, I already using direct IP to connect. Should I increase or decrease the bit rate to achieve it?

**Reply by xlionjuan** [reply]
It shouldn't matter, but if your FPS is perfectly staying at 60, maybe check the settings?

Or change the settings to something else and then change back.
